src/tcg-to-deckbox.py

```diff
@@ -178,12 +181,14 @@ def getPathPrefix():
         if row["Name"] in ixalan_bab and row["Edition"] == "Buy-A-Box Promos":
             row["Edition"] = "Black Friday Treasure Chest Promos"
             skip_scryfall_names=True
-
+        # Handle other BaB promo cards
+        if row["Name"] in bab_mapping and row["Edition"] == "Buy-A-Box Promos":
+            row["Edition"] = bab_mapping[row["Name"]]
             
         # For BFZ lands...there's no differentiator from the full arts and the non full arts.
         row["Name"] = row["Name"].replace(" - Full Art", "")
 
-        row["Name"] = re.sub(r" \([^)(]+\)$", "", row["Name"])
+        row["Name"] = re.sub(r" \(.*\)", "", row["Name"])
```

Comment:
adjusted as I have some cards that tcgplay spits out with multiple groups of parens, e.g. "Blue Sun's Zenith (JP Alternate Art) (Foil Etched)"